guides/equality-diversity-and-inclusion/policy.md

```diff
@@ -0,0 +1,89 @@
+# Equality, Diversity & Inclusion Policy
+
+## About this policy
+
+This policy applies to everyone who works for Made Tech, or who acts on Made Tech’s behalf. All team members have a role in promoting equality, diversity and inclusion at work. We all have a personal responsibility to comply with the policy and to ensure, as far as possible, that others do the same.
+
+Made Tech is responsible for this policy, and for ensuring that all our staff understand their rights and obligations as detailed within it, and for any necessary training on equal opportunities.
+
+This policy does not form part of your contract of employment, and we may amend it at any time.
+
+## The legal framework
+
+It is illegal to discriminate against a person on the basis of the following [Protected Characteristics](https://www.gov.uk/discrimination-your-rights):
+
+- Age
+- Disability
+- Gender reassignment
+- Marital or civil partner status
+- Pregnancy or maternity
+- Race (including colour, nationality, ethnic or national origin)
+- Religion, religious belief or similar philosophical belief
+- Sex
+- Sexual orientation (ie homosexuality, bisexuality or heterosexuality)
```

Comment:
Do we really need to list out all the references from the external source? Given that we aim to go above and beyond this, can we not just have the link to the external source and use the end of 25?

Reply:
It's a valid question, but I think they should be here so that the flow of reading this document is not broken by having to go somewhere else.